client/src/components/BarGraph.jsx

```diff
@@ -1,58 +1,76 @@
 import React from 'react';
 import styled from 'styled-components';
-import fonts from './Fonts.js';
-import {margins} from './Constants.jsx';
+import {margins, Text, Fonts, animation} from './Constants.jsx';
```

Comment:
Nice code reuse, lots of Constants I  see

Reply:
we out here